src/readonly/Map.hsig

```diff
@@ -0,0 +1,20 @@
+signature Map
+          ( -- * Types
+            Map
+          , Key
+
+
+
+          ) where
+
+
+
+data Map k v
+class Key k
+
+
+
+
+
+
+
```

Comment:
Is this a generated file?

Reply:
No, it's not generated. There were two signatures before: for readonly containers and for modifiable containers. But this doesn't work for some reason in this new solution. So I have only one file now, but haven't deleted old files yet. Will cleanup later :+1: